doc/examples/aseprite/README.md

```diff
@@ -0,0 +1,16 @@
+# aseprite
+
+A new Flutter project.
```

Comment:
How about improving this readme for the example project? Just a single phrase to describe it is enough!

Reply:
Done